### ISSUE TITLE
[IDEA] Better CSS for nested ordered lists

### ISSUE THREAD

**Issue by saqimtiaz** [TiddlyWiki/TiddlyWiki5#5186]
([via google group user request](https://groups.google.com/g/tiddlywiki/c/ZWwNlhR-9N8)) 

We should consider improving the styling of nested ordered lists, or at least include in the core a class or two that can be used to switch the presentation without end-users needing to write their own CSS.

Currently nested ordered lists present as 
![image](https://user-images.githubusercontent.com/68092/100940160-c153e780-34f7-11eb-9797-0f99817c7e9d.png)

Alternatives:
![image](https://user-images.githubusercontent.com/68092/100940206-d0d33080-34f7-11eb-9e80-d09b40cf9bc3.png)
or
![image](https://user-images.githubusercontent.com/68092/100940220-d597e480-34f7-11eb-99fa-3139176211c8.png)

**Comment by Jermolene** [comment]
To make it discoverable, perhaps we should have a dropdown under theme tweaks?

**Comment by saqimtiaz** [comment]
@Jermolene yes indeed, that is a good idea. A dropdown that changes the default styling of ordered lists would be an excellent way to go about it.

**Comment by saqimtiaz** [comment]
@BurningTreeC any chance you might be able to look at this? Would be great to get it into 5.1.23 as I believe it's a small change that can make a significant usability difference.

**Comment by BurningTreeC** [comment]
@saqimtiaz I need to have a look how the CSS part needs to be done, but I can try

**Comment by saqimtiaz** [comment]
@BurningTreeC this might be of help:
https://support.knowledgeowl.com/help/customize-nested-numbered-list-styles
https://stackoverflow.com/questions/10405945/html-ordered-list-1-1-1-2-nested-counters-and-scope-not-working

**Comment by BurningTreeC** [comment]
Thank you @saqimtiaz , I will have a look this evening

**Comment by saqimtiaz** [comment]
The need for this came up again today, so bumping it in the hopes that one of us has the time to tackle it soon.

**Comment by kookma** [comment]
> To make it discoverable, perhaps we should have a dropdown under theme tweaks?

Do you think a dropdown for ordered list and also for headings?

What is the requirements:

- only outline list?
- have option to use different types? e.g alphabetic, numeric, roman, ...
